src/bin/FacilitatorStart.ts

```diff
@@ -0,0 +1,168 @@
+import { Config, FacilitatorConfig } from '../Config';
+import MosaicConfig from '../MosaicConfig';
+import { FacilitatorStartException } from '../Exception';
+
+/**
+ * It parses and evaluates input params of facilitator start command to get config object.
+ */
+export default class FacilitatorStart {
```

Comment:
I am not convinced with FacilitatorStart  class name. The class validates, parses input parameters and returns config and mosaic object. Class naming can be improved. Not able to suggest a better name now 🤔